cfme/utils/appliance/__init__.py

```diff
@@ -1453,12 +1453,6 @@ def add_product_repo(self, repo_url, **kwargs):
                 # It is already there, so just enable it
                 self.enable_disable_repo(repo_id, True)
                 return repo_id
-        product, ver = match.groups()
-        repos = self.find_product_repos()
-        if product in repos:
-            for v, i in repos[product].items():
-                logger.info("Deleting %s repo with version %s (%s)", product, v, i)
-                self.ssh_client.run_command("rm -f /etc/yum.repos.d/{}.repo".format(i))
```

Comment:
RHEL 8 upgrades require both BaseOS and Appstream repos to be enabled, this code is removing one of them. I do not believe this is needed. We should start with a clean appliance for a rhel upgrade